examples/favorites_advanced/favorites_advanced_base/lib/src/repositories/puppies_repository.dart

```diff
@@ -19,7 +19,7 @@ class PuppiesRepository {
   final _noInternetConnectionErrorString =
       'No internet connection. Please check your settings.';
 
-  /// Simulate delays of the API http requests
+  /// Simuof the API http requests
```

Comment:
@StanislavPrime, please revert the previous state of the comment.

Reply:
The comment was reverted to /// Simulate delays of the API http requests